src/System.IO.IsolatedStorage/tests/System/IO/IsolatedStorage/IsoStorageTest.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace System.IO.IsolatedStorage
 {
+    [Collection("Store collection")]
```

Comment:
To cause the fixture to get applied that will clean the stores when all of the tests run in this collection. (And for sanity, clean before starting.)